gulpfile.js

```diff
@@ -127,6 +128,13 @@ gulp.task('styles', function stylesTask() {
     .pipe(gulp.dest(DEST_PATH + 'styles'));
 });
 
+gulp.task('sass-lint', function() {
```

Comment:
Tis but a warning, but you could just give this anon-func a name like `sassLintTask()`.

```
gulpfile.js
  131:24  warning  Missing function expression name  func-names

✖ 1 problem (0 errors, 1 warning)
```